tests/templates/kuttl/smoke-nodeport/02-checks-container.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: checks
+  labels:
+    app: checks
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: checks
+  template:
+    metadata:
+      labels:
+        app: checks
+    spec:
+      containers:
+        - name: checks
+          image: docker.stackable.tech/stackable/testing-tools:0.2.0-stackable0.0.0-dev
+          command: ["sleep", "infinity"]
```

Comment:
Why not just run this as a `Job` rather than go through this dance?

Reply:
replaced with a Job